### DISCUSSION TITLE
Part 2 of Pinterest Firebase: InnerHTML of null error

### DISCUSSION THREAD

**Discussion by elliewalkersm** [nss-evening-cohort-14/evening-client-side#160] [Help Me!] [answered]
Ellie Walker
Searched other tickets: Yes

Issue: It appears that the cards for my boards are loading before my domBuilder, however, I have my domBuilder called first in my startApp function.

errors: "
boards.js?c85b:2 Uncaught (in promise) TypeError: Cannot set property 'innerHTML' of null
    at showBoards (boards.js?c85b:2)
    at eval (startApp.js?a45b:15)"

Google links: 
-https://teamtreehouse.com/community/uncaught-typeerror-cannot-set-property-innerhtml-of-null-at-print-scriptjs29-at-scriptjs32
-https://stackoverflow.com/questions/18239430/cannot-set-property-innerhtml-of-null
https://discuss.codecademy.com/t/why-this-isnt-working-whats-causing-error/63416

What I have tried:
- commenting out everything except what is needed to show the boards on page load
- calling domBuilder on the auth page along with startApp

Here are my files: 
-https://github.com/elliewalkersm/ASSIGNMENT-Firebase-Pinterest/blob/create-boards/src/javascripts/components/boards.js
-https://github.com/elliewalkersm/ASSIGNMENT-Firebase-Pinterest/blob/create-boards/src/javascripts/components/domBuilder.js
-https://github.com/elliewalkersm/ASSIGNMENT-Firebase-Pinterest/blob/create-boards/src/javascripts/views/startApp.js

**Comment by elliewalkersm** [comment, marked as answer]
I was targeting the #app div for my navigationBar. Once I changed that to target #navigation within my domBuilder, it worked!

**Reply by drteresavasquez** [reply]
@elliewalkersm that was not quite the issue. Did you have an import in main.js that was giving you an issue? What was that?

**Reply by elliewalkersm** [reply]
Yes! I also was calling my navigationBar in main.js when I was trying to trouble shoot and never deleted it!
